### ISSUE TITLE
Trained vehicle tends to freeze at the entrance of roundabout env 

### ISSUE THREAD

**Issue by nomatterhoe** [eleurent/highway-env#99]
Hi, @eleurent,  recently I've been exploring your excellent project. I used PPO2 from stable-baselines to train vehicle in roundabout env, one weird phenomena I noticed was that the trained vehicle tends to freeze at the entrance of roundabout (not move anymore). This happened quite often when I evaluated the trained model. I'm not sure whether it's the problem of  hyperparameters but I didn't make much changes to hyperparameters. Do you have any idea why this happens and suggestion on how to solve this? 

My testing code was like this:

```
env = gym.make("roundabout-v0")
model = PPO2(MlpPolicy, env, verbose=1, learning_rate=lambda f: f * 5.0e-4, cliprange=lambda f: f * 0.1)
model.learn(total_timesteps=int(1e5))
model.save('ppo2_mlp_roundabout')
```

Thanks for your help.

**Comment by eleurent** [comment]
Hi @nomatterhoe,
Thank you for that feedback, that's interesting.
I think that for this roundabout environment, I actually have never tried model-free algorithms so far, only model-based (planning) approaches. I'll try and see if I can reproduce the conservative behaviours that you get.

Hypothesis: the model has trouble figuring out whether or not a vehicle is coming, and settles for the safe option.
This could be alleviated by a different choice of observation type and neural network architecture, see e.g.[this work](https://eleurent.github.io/social-attention/) on the intersection environment.

**Comment by eleurent** [comment]
That being said, I just noticed that the default observation for roundabout is the Time To Collision observation, which is not appropriate since it was designed for straight roads.
I just changed it to a Kinematics observation, which is better suited (OccupancyGrid could work as well)

**Comment by nomatterhoe** [comment]
Thanks for your reply @eleurent , following your suggestion, I tried the Roundabout env with OccupancyGrid observation and Kinematics observation, the freeze problem was solved in both cases. But it seems like the training curve won't converge (following is the roundabout env with Kinematics observation trained using ppo2)

![image](https://user-images.githubusercontent.com/4505336/90706109-c76ee100-e2d7-11ea-9320-0f209d19e338.png)

The training curves also can't converge in Intersection env when I trained the vehicle using dqn and ppo2, I tried adjusting learning rate and total_timesteps, didn't make differences. I had a look at your [paper](https://eleurent.github.io/social-attention/), it seems like you customized the DQN algorithm for the intersection env by adding attention layers in your network architecture. So my new question is, in your opinion, is it possible at all that a baseline algorithm can perform/learn well under these complex driving environments such as roundabout and intersection?
